pkg/core/core_test.go

```diff
@@ -181,3 +181,8 @@ func TestContainsAddress(t *testing.T) {
 		})
 	}
 }
+
+func TestParseCIDRError(t *testing.T) {
```

Comment:
Is it possible to write it in the same style as the other tests? Otherwise, could you please rename it to `TestParseCIDRWithEmptyStringError` or something along those lines

Reply:
Yep, that's not a problem. I'll do that tomorrow :)